packages/stark-ui/assets/styles/components/_button-theme.scss

```diff
@@ -23,7 +23,7 @@
       background-color: rgba($color: $contrast, $alpha: 0.1);
     }
   }
-  .mat-icon-button {
+  &.mat-icon-button {
```

Comment:
Are we sure of this change? `.xxx` is not the same as `&.xxx`

Reply:
Actually, the rule was incorrect before. The `mat-icon-class` is added on the same element than the `mat-button` class. It's a fix 😊